IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayLevelCrossing/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayLevelCrossing" UniqueId="7f547f91-6dd0-161c-9824-1f608dd660d9" ApplicableType="IfcFacilityPart/IfcFacilityPartCommonTypeEnum(.LEVELCROSSING.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayLevelCrossing" UniqueId="7f547f91-6dd0-161c-9824-1f608dd660d9" ApplicableType="IfcFacilityPart/IfcFacilityPartCommonTypeEnum(.LEVELCROSSING.)" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
This ApplicableType-string would probably change after the redesign of facility part structure.